doc/articles/silverlight-migration-landing.md

```diff
@@ -0,0 +1,32 @@
+# Migrating from Silverlight to WinUI and Uno Platform Documentation
+
+Silverlight will reach the end of support  on October 12, 2021. As luck would have it, a new Windows UI framework is about to RTM in March 2021 – WinUI – the modern native UI platform of Windows. And with WinUI launch there is a renewed desire by C# and XAML developers to write single codebase applications for Windows and the Web.
+
+Enter Uno Platform. Hello “Rich Internet Applications (RIA)” again!
+
+There are 47 pages of Silverlight migration documentation to help you with migration – accompanying source code, techniques and considerations you need to make when migrating your Silverlight application.
```

Comment:
It's a lot but not quite that much :D More like 18 pages, right?

I just pinged @sasakrsmanovic to correct this in the blog post as well

Reply:
Articles vs. screens?